VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
-1, I would say let it be part of Makefile instead of creating another file just for the version